examples/simple_example.rs

```diff
@@ -48,12 +48,19 @@ fn main() {
     let mut res = 0;
     let pid = unsafe { getpid() };
 
-    let mut tracer = Tracer::new().trace_filename("simple_example.ptt");
-    tracer.start_tracing();
+    let mut tracer = Tracer::new()
+                            .trace_filename("simple_example.ptt")
+                            .unwrap_or_else(|e| panic!(format!("{}", e)));
```

Comment:
What's the difference between this line and just saying `unwrap()`?

Reply:
Ah, does `unwrap()` print the error message then? If so, we can just unwrap.

Reply:
It does. I pushed a change to simply unwrap.